netebpfext/net_ebpf_ext_bind.c

```diff
@@ -117,10 +118,35 @@ _net_ebpf_extension_bind_on_client_detach(_In_ const net_ebpf_extension_hook_cli
 // NMR Registration Helper Routines.
 //
 
+static NTSTATUS
+_net_ebpf_bind_update_registry_entries()
```

Comment:
if an extension has to do this for every program_info and hook; just prior to the NMR registration and the NMR registration process itself does not change, why cant eBPF core do it on behalf of the extensions as part of provider attach callback?

Reply:
core does not have all the information today (like section prefixes, etc.). So it can only update a part of the store info.